frontend2/src/views/Scrimmaging.tsx

```diff
@@ -0,0 +1,340 @@
+              onPage={(page) => {
+                setQueryParams({ ...queryParams, teamsPage: page.toString() });
+              }}
+            />
+          }
+        />
+      </div>
+
+      <div className="justify-right mb-1 flex flex-row space-x-4">
+        <h1 className="mb-2 text-2xl font-bold leading-7 text-gray-900">
+          Scrimmage History
+        </h1>
+        <Button
+          disabled={scrimsLoading}
+          label="Refresh!"
+          variant="dark"
+          onClick={() => {
+            setQueryParams({ ...queryParams, scrimsPage: "1" });
+            if (scrimsPage === 1 && !scrimsLoading) {
+              void fetchScrimsData();
+            }
+          }}
+        />
+      </div>
+      <div className="mb-4 w-5/6">
+        <BattlecodeTable
+          data={scrimsData?.results ?? []}
+          loading={scrimsLoading}
+          columns={[
+            {
+              header: "Score",
+              value: (match) => <MyMatchResult match={match} />,
+            },
+            {
+              header: "Δ",
+              value: (match) => "TODO",
+            },
+            {
+              header: "Opponent (Δ)",
+              value: (match) => "TODO",
+            },
+            {
+              header: "Ranked",
+              value: (match) => (match.is_ranked ? "Ranked" : "Unranked"),
+            },
+            {
+              header: "Status",
+              value: (match) => "TODO",
+            },
+            {
+              header: "Replay",
+              value: (match) => "TODO",
+            },
+            {
+              header: "Created At",
+              value: (match) => new Date(match.created).toLocaleDateString(),
+            },
+          ]}
+          bottomElement={
+            <BattlecodeTableBottomElement
+              totalCount={scrimsData?.count ?? 0}
+              pageSize={10}
+              currentPage={scrimsPage}
+              onPage={(page) => {
+                setQueryParams({ ...queryParams, scrimsPage: page.toString() });
+              }}
+            />
+          }
+        />
+      </div>
+
+      <div className="justify-right mb-1 flex flex-row space-x-4">
+        <h1 className="mb-2 text-2xl font-bold leading-7 text-gray-900">
+          Recent Tournament Matches
+        </h1>
+        <Button
+          disabled={tourneyLoading}
+          label="Refresh!"
+          variant="dark"
+          onClick={() => {
+            setQueryParams({ ...queryParams, tourneyPage: "1" });
+            if (tourneyPage === 1 && !tourneyLoading) {
+              void fetchTourneyData();
+            }
+          }}
+        />
+      </div>
+
+      <div className="mb-4 w-5/6">
+        <BattlecodeTable
+          data={tourneyData?.results ?? []}
+          loading={tourneyLoading}
+          columns={[]}
+          bottomElement={
+            <BattlecodeTableBottomElement
+              totalCount={tourneyData?.count ?? 0}
+              pageSize={10}
```

Comment:
should make this an optional prop in `BattlecodeTableBottomElement.tsx` and default to 10. Maybe even don't allow this to be changed and add page sizing to be changed in db calls in a future PR?

Reply:
TODO in future PR